requirements.txt

```diff
@@ -1,5 +1,2 @@
 galaxy_importer
 packaging
-pulpcore>=3.0
```

Comment:
We still need to depend on pulpcore.

What about these other deps why are they being removed?

Reply:
old version of release script